### ISSUE TITLE
Superfluous "ok" output for dot-commands

### ISSUE THREAD

**Issue by revolter** [sqlcipher/sqlcipher#353]
### Actual Behavior

I think I indirectly updated the sqlcipher version on my system using `brew`, and now `.dump` and `.output` commands are printing out "ok\n" before the actual output:

```bash
> sqlcipher -cmd "PRAGMA key = '<password>'; PRAGMA cipher_compatibility = 3; PRAGMA user_version" database.sqlite .output
ok
5
```

### Steps to Reproduce

SQLCipher version: `3.30.1 2019-10-10 20:19:45 18db032d058f1436ce3dea84081f4ee5a0f2259ad97301d43c426bc7f3dfalt1`

*Note:* If you are not posting a specific issue for the SQLCipher library, please consider posting your question to the SQLCipher [discuss site](https://discuss.zetetic.net/c/sqlcipher).  Thanks!

**Comment by revolter** [comment]
Off-topic issue:

```bash
> brew upgrade sqlcipher
Warning: sqlcipher 4.3.0 already installed
```

So am I using `3.30.1` or `4.3.0`? I don't understand...

**Comment by developernotes** [comment]
Hi @revolter 

The output of the `ok` value was a change introduced in upstream SQLite. Those changed were merged into the SQLCipher 4.3.0 release.

**Comment by developernotes** [comment]
>So am I using 3.30.1 or 4.3.0? I don't understand...

The SQLite version is 3.30.1, SQLCipher is 4.3.0.
